platformics/codegen/templates/graphql_api/types/class_name.py.j2

```diff
@@ -257,16 +244,17 @@ class {{ cls.name }}(EntityInterface):
     {{ attr.name }}: {{ getType("bool", attr.required) }}
         {%- elif attr.type == "date" %}
     {{ attr.name }}: {{ getType("datetime.datetime", attr.required) }}
-        {%- elif attr.type == "File" %}
-    {{ attr.name }}_id: Optional[strawberry.ID]
-    {{ attr.name }}: Optional[Annotated["File", strawberry.lazy("platformics.graphql_api.files")]] = load_files_from("{{ attr.name }}")  # type: ignore
         {%- elif attr.type == cls.name %}
     {{ attr.name }}_id: Optional[strawberry.ID]
-        {%- elif attr.inverse %}
+        {%- elif attr.is_entity %}
```

Comment:
I also am curious about the inverse -> is_entity changes

Reply:
In a lot of places in our codegen templates we've been using the `attr.inverse` property to decide whether a given field is a relationship to another type. I'm not entirely sure why -- maybe we wrote these templates before we had an `is_entity` property? However, if *all* relationships include a back-reference from the related class, that works fine. But with the relationship between SequencingReads <-> Files, the Files type doesn't have a back reference to the SequencingReads type. I want to support relationships without backrefs in general, so I'm updating the templates to avoid using `attr.inverse` except where it's actually needed.

Reply:
Ah, so we aren't actually using the `inverse` property, we were just using that as a heuristic to see whether the attribute is another entity that we're defining?

Reply:
Yep, exactly. :'(